pkg/controller/resourcequota/resource_quota_controller_test.go

```diff
@@ -370,12 +370,12 @@ func TestAddQuota(t *testing.T) {
 				},
 				Spec: v1.ResourceQuotaSpec{
 					Hard: v1.ResourceList{
-						"count/foobars.example.com": resource.MustParse("4"),
+						"foobars.example.com": resource.MustParse("4"),
 					},
 				},
 				Status: v1.ResourceQuotaStatus{
 					Hard: v1.ResourceList{
-						"count/foobars.example.com": resource.MustParse("4"),
+						"foobars.example.com": resource.MustParse("4"),
```

Comment:
Do we want to add a similar test case for "requests.foobars.example.com" but with "expectedPriority: true"?

Reply:
That makes sense, would add one

Reply:
Case added